### ISSUE TITLE
College profiles - title attr on fact icons should be removed

### ISSUE THREAD

**Issue by cjg89** [UCF/Main-Site-Theme#258]
**Describe the bug**

See title.

**Steps to Reproduce**

https://github.com/UCF/Main-Site-Theme/blob/master/taxonomy-colleges.php#L56

**Expected Behavior**

These images should not utilize a `title` attribute, so that the image `alt` attribute can be empty (marking it as decorative) without accessibility scans throwing errors.

**Possible Fix**

Remove the attr.

**Your environment:**

n/a

**Additional context**

n/a
